vendor/composer/autoload_real.php

```diff
@@ -22,6 +22,8 @@ public static function getLoader()
             return self::$loader;
         }
 
+        require __DIR__ . '/platform_check.php';
```

Comment:
I saw that you have removed this platform checker before, therefore I would assume that adding it here is part of the plan? Right?

Reply:
this is auto-generatoed by composer, unfortunately it seems to depend on which version of composer the classmap is generated with.